test/acl.test.js

```diff
@@ -48,22 +44,28 @@ describe('security scopes', function() {
     testModel.attachTo(ds);
   });
 
-  it('should allow access to models for the given scope by wildcard', function() {
+  it('should allow access to models for the given scope by wildcard', function(done) {
     Scope.create({name: 'userScope', description: 'access user information'},
       function(err, scope) {
         ACL.create({
           principalType: ACL.SCOPE, principalId: scope.id,
           model: 'User', property: ACL.ALL,
           accessType: ACL.ALL, permission: ACL.ALLOW,
         }, function(err, resource) {
-          Scope.checkPermission('userScope', 'User', ACL.ALL, ACL.ALL, checkResult);
-          Scope.checkPermission('userScope', 'User', 'name', ACL.ALL, checkResult);
-          Scope.checkPermission('userScope', 'User', 'name', ACL.READ, checkResult);
+          async.parallel([
+            clbk => Scope.checkPermission('userScope', 'User', ACL.ALL, ACL.ALL, clbk),
```

Comment:
I find `clbk` as rather cryptic. Could you please use `cb`, `callback` or `next` instead?